service/src/main/resources/api/service_openapi.yaml

```diff
@@ -157,6 +157,25 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/workspaces/v1/{workspaceId}/gcp/enablepet:
```

Comment:
I think this should be clearly marked as a temporary endpoint with no promise of ongoing support. We do not want this as part of the WSM model in the long term, do we?

IMHO, this is another case where Sam cloud sync does not support the WSM requirements. If we were notified as new readers and writers are added to the workspace, then we would be able to set this attribute this ourselves without an explicit request.

Reply:
Agreed that this is temporary - added a TODO and a note for swagger UI.